docs/testing/dotnet_testing_selenium.md

```diff
@@ -73,6 +73,8 @@ In root (*.sln) directory, build the solution using: ) the solution:
 dotnet test
 ```
 
+As a result of the test execution - the 'BDDfy.html' test report will be automatically generated.
```

Comment:
As a result of test execution - a 'BDDfy.html' test report will be automatically generated.